cloudanalysis/cloudanalysis_fv3driver.f90

```diff
@@ -796,6 +797,7 @@ program cloudanalysis
 !
 !  call MPI_Barrier(mpi_comm_world, ierror)
   do k=1,nsig
+     kk=nsig+1-k
```

Comment:
The phy_data file uses vertical levels from bottom to top, opposite of the other intermediate netcdf files and the expected direction in the Fortran code.  I use the kk index to reverse the direction; if there is a better way to do it I can make that change.

Reply:
I'd say this GitHub comment would make a fantastic code comment.

Reply:
thanks for the idea.  Done.